onnxmltools/convert/keras/operator_converters/__init__.py

```diff
@@ -26,4 +26,4 @@
 from . import SimpleRNN
 from . import Upsample
 from . import ZeroPad
-
+from . import Dropout
```

Comment:
Move it right after `from . import Dot` please.

Reply:
switched